recipes-omnect/initrdscripts/omnect-os-initramfs/grub-sh

```diff
@@ -46,9 +46,11 @@ save_fsck_status()
     if [ ! "${part_name}" = "boot" ]; then
         test_and_mount_boot || return 1
         gzip -9c ${source_file} | base64 -w0 - > ${ROOTFS_DIR}/boot/fsck.${part_name}
+        run_cmd umount ${ROOTFS_DIR}/boot
     elif [ ! "${fsck_res}" = "2" ]; then
         test_and_mount_boot || return 1
         set_bootloader_env_var omnect_fsck_boot $(gzip -9c ${source_file} | base64 -w0 -) || set_bootloader_env_var omnect_fsck_boot $(echo "fsck output to big" | gzip - | base64 -w0 -)
+        run_cmd umount ${ROOTFS_DIR}/boot
```

Comment:
imho, you should do it, where the error happened ... and add a comment why we need the unmount